src/io/stun.rs

```diff
@@ -32,16 +32,16 @@ pub fn stun_resend_delay(send_count: usize) -> Duration {
     Duration::from_millis(capped)
 }
 
+/// Possible errors when handling STUN messages.
 #[derive(Debug, Error)]
 pub enum StunError {
+    /// A STUN message could not be parsed.
     #[error("STUN parse error: {0}")]
     Parse(String),
 
+    /// An IO error occurred while handling a STUN message.
     #[error("STUN io: {0}")]
     Io(#[from] io::Error),
-
-    #[error("STUN error: {0}")]
-    Other(String),
```

Comment:
Extracted removal of this into a dedicated PR: https://github.com/algesten/str0m/pull/421.